airbyte-integrations/connectors/source-oracle/build.gradle

```diff
@@ -0,0 +1,32 @@
+plugins {
+    id 'application'
+    id 'airbyte-docker'
+    id 'airbyte-integration-test-java'
+}
+
+application {
+    mainClass = 'io.airbyte.integrations.source.oracle.OracleSource'
+}
+
+dependencies {
+
+    implementation group: 'xerces', name: 'xercesImpl', version: '2.12.1'
```

Comment:
```suggestion
    // required so that log4j uses a standard xml parser instead of an oracle one (that gets pulled in by the oracle driver)
    implementation group: 'xerces', name: 'xercesImpl', version: '2.12.1'
```